packages/cli/lib/usageTracking.js

```diff
@@ -7,6 +7,25 @@ const { version } = require('../package.json');
 const { getPlatform } = require('./environment');
 const { setLogLevel } = require('./commonOpts');
 
+/* **
+Available tracking meta properties
+
+  - action: "The specific action taken in the CLI"
+  - os: "The user's OS"
+  - nodeVersion: "The user's version of node.js"
+  - nodeMajorVersion: "The user's major version of node.js"
+  - version: "The user's version of the CLI"
+  - command: "The specific command that the user ran in this interaction"
+  - authType: "The configured auth type the user has for the CLI"
+  - step: "The specific step in the auth process"
+  - assetType: "The CMS asset type"
+  - mode: "The CLI mode (draft or publish"
+  - type: "The upload type (file or folder)"
+  - file: "Whether or not the 'file' flag was used"
+  - successful: "Whether or not the CLI interaction was successful"
+
+*/
+
```

Comment:
I figure this could be useful to know what properties are available to pass through to the tracker, let me know what you think!

Reply:
This is great, thanks!